src/ngraph/op/experimental/generate_mask.hpp

```diff
@@ -47,23 +59,29 @@ namespace ngraph
             virtual std::shared_ptr<Node>
                 copy_with_new_args(const NodeVector& new_args) const override;
 
+            const element::Type& get_element_type() const { return m_element_type; }
+            void set_element_type(const element::Type& element_type)
+            {
+                m_element_type = element_type;
+            }
+
+            /// Deprecated accessor for transitional attributes
+            const Shape& get_shape() const { return m_shape; }
```

Comment:
This clashes with `Node::get_shape()`, which seems dangerous.

Reply:
renamed to get_mask_shape() in new commit.